R/zzz.R

```diff
@@ -1,4 +1,22 @@
-.lockbox_env <- new.env()
+.lockbox_env <- new.env(parent = emptyenv())
+
+set_session_id <- function() {
+  if (is.null(.lockbox_env$session_id)) {
+    ## This is our first run, so we want to register the finalizer to drop our temp directories
+    reg.finalizer(.lockbox_env, function(env) {
+      try(unlink(normalizePath(paste0("~/.R/", env$session_id), mustWork = FALSE), recursive = TRUE), silent = TRUE)
+      try(unlink(normalizePath(paste0("~/.R/", env$session_id, "_staging"), mustWork = FALSE), recursive = TRUE), silent = TRUE)
+    }, onexit = TRUE)  
+  }
+  
+  if (requireNamespace("uuid", quietly = TRUE)) {
+    .lockbox_env$session_id <- .lockbox_env$session_id %||% uuid::UUIDgenerate()
+  } else {
+    message("Install package::uuid for decreased risk of ")
```

Comment:
@abelcastilloavant Thanks for catching that.  It was an incomplete thought.  I'd decided not to send a message there.  I'll drop that message.